_episodes/06-blurring.md

```diff
@@ -254,39 +267,38 @@ also be very familiar to you at this point.
 
 ~~~
 # read and display original image
-image = cv2.imread(filename = filename)
-cv2.namedWindow(winname = "original", flags = cv2.WINDOW_NORMAL)
-cv2.imshow(winname = "original", mat = image)
-cv2.waitKey(delay = 0)
+image = skimage.io.imread(fname=filename)
+viewer = ImageViewer(image)
+viewer.show()
 ~~~
 {: .python}
 
 Now we apply the average blur:
 
 ~~~
 # apply Gaussian blur, creating a new image
-blurred = cv2.GaussianBlur(src = image, 
-    ksize = (k, k), sigmaX = 0)
+blurred = skimage.filters.gaussian(
+    image, sigma=(sigma, sigma), truncate=3.5)
 ~~~
 {: .python}
 
-The first two parameters to `cv2.GaussianBlur()` are the image to blur, 
-`image`, and a tuple describing the shape of the kernel, `(k, k)`. The third 
-parameter is the standard deviation for the two-dimensional Gaussian 
-distribution in the x dimension. If we pass in `0`, as we have done here, 
-OpenCV automatically determines default standard deviations for both the x and 
-y dimensions, based on the kernel size. This is how we will normally invoke the
-`cv2.GaussianBlur()` function. The `cv2.GaussianBlur()` function returns a new 
-image after the filter has been applied.
-
+The first two parameters to `skimage.filters.gaussian()` are the image to blur,
+`image`, and a tuple defining the sigma to use in y- and x-direction,
+`(sigma, sigma)`. The third parameter `truncate` gives the radius of the kernel
+in terms of sigmas. A Gaussian is defined from -infinity to +infinity. A
+discrete Gaussian can only approximate the real function. The `truncate`
+parameter steers at what distance to the center of the function it is not
+approximated any more. In the above example we set `truncate` to 3.5. With a
+`sigma` of 1.0 the resulting kernel size would be 7.
+The last parameter is to tell skimage to interpret our image, that has three
+dimensions, as a multichannel color image.
```

Comment:
This does not fit to the code example; you only give three parameters:
`image`, `sigma` and `truncate`, but the explanation talks about 4 parameters.